nodejs/week1/homework/src/backend/routes/meal.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("../data/meals.json");
+const reviews = require("../data/reviews.json");
+
+router.get("/meal", (request, response) => {
+    const randomMeal = Math.floor(Math.random() * meals.length + 0);
+
+    meals.forEach((meal) => {
+        meals.filter((review) => review.id === meal.id);
+    });
+    response.send(randomMeal);
```

Comment:
I think you forgot to index `meals` with this random index and return an object... ☝️